app/classifier/index.cjsx

```diff
@@ -49,10 +75,15 @@ Classifier = React.createClass
       @loadSubject subject
     if nextProps.classification isnt @props.classification
       @prepareToClassify nextProps.classification
+    @context.geordi?.remember subjectID: @props.subject?.id
```

Comment:
This will remember the current props subject id, not the nextProps... since this is in `componentWillReceiveProps` and there is no check happening, this is running every time new props is received and remembering current props. Not sure what the goal is here, but you might want `nextProps.subject?.id` instead after a conditional that checks `if @props.subject isnt nextProps.subject`